extensions/wikia/Insights/Insights.i18n.php

```diff
@@ -188,7 +162,7 @@
 	'insights-list-description-flags' => 'Verwende Markierungen, um Artikelseiten zu organisieren und zu beschriften. Diese Insight-Liste ermöglicht es dir, benutzerdefinierte Berichte zu markierten Artikeln anzusehen, um so die Artikel zu finden, bei denen es am dringendsten ist, dass Verbesserungen vorgenommen werden. [[Hilfe:Flags|Mehr über Markierungen erfahren]]',
 	'insights-notification-message-inprogress-flags' => 'Diese Seite hat eine Markierung.',
 	'insights-list-subtitle-nonportableinfoboxes' => 'Nicht-portable Infoboxen',
-	'insights-list-description-nonportableinfoboxes' => 'Die folgenden Infoboxen werden mit Tabellen oder Divs erstellt und sollten umgewandelt werden, damit sie das neue [[Hilfe:PortableInfoboxes|Infobox-Markup]] verwenden und so für alle Besucher korrekt gerendert werden. Wenn du nicht mehr weiter kommst, kannst du auch [[w:c:de.community:Diskussionsforum:Support-Anfragen:_Technisches|unser technisches Forum besuchen und dort Fragen stellen]].',
+	'insights-list-description-nonportableinfoboxes' => 'Die folgenden Infoboxen werden mit Tabellen oder Divs erstellt und sollten umgewandelt werden, damit sie das neue [[Hilfe:PortableInfoboxes|Infobox-Format] verwenden und so für alle Besucher korrekt angezeigt werden. Wenn du nicht mehr weiter kommst, kannst du auch [[w:c:de.community:Diskussionsforum:Support-Anfragen:_Technisches|unser technisches Forum besuchen und dort Fragen stellen]].',
```

Comment:
`[[Hilfe:PortableInfoboxes|Infobox-Format]` <-- this will be a broken link and needs an extra closing square bracket.